TimeTable/Models/BussinessModels/Projects/ProjectTag.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  ProjectTag.swift
+//  TimeTable
+//
+//  Created by Piotr Pawluś on 29/05/2019.
+//  Copyright © 2019 Railwaymen. All rights reserved.
+//
+
+import UIKit
+
+enum ProjectTag: String, Codable {
+    case development = "dev"
+    case internalMeeting = "im"
+    case clientCommunication = "cc"
+    case research = "res"
+    
+    var lozalized: String {
+        return self.rawValue.localized
```

Comment:
Is it good to have 2-3 letter localization key? Add at least some prefix like `project_tag`